src/main/java/com/google/api/codegen/viewmodel/SimpleInitCodeLineView.java

```diff
@@ -31,8 +31,8 @@
 
   public abstract List<String> doc();
 
-  /** @return if this is the first item in a list of lines */
-  public abstract boolean isFirst();
+  /** @return if a newline has been added above this line */
```

Comment:
This is actually the opposite of what I had intended :P 

Optional: I'd prefer formulating this as "if a newline should be added above this line" because it makes the property about this object, and not about its context. I recognize that that might be a distinction without a difference given how it's used in the transformer.

Reply:
Sounds the same to me. I liked isFirst because not all languages will want a newline here (ruby doesn't python does), but we can revisit the name when we do the others.